### PR TITLE
Adding pyemcee

### PR DESCRIPTION
**Pull request by danehkar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by danehkar** [comment]
@conda-forge/staged-recipes ready for review!

File: recipes/pyemcee/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - pip
+    - python >=3.4
+    - numpy >=1.15
+    - scipy >=1.2
+    - matplotlib >=2.2
+  run:
+    - python >=3.4
+    - numpy >=1.15
+    - scipy >=1.2
+    - matplotlib >=2.2
```
Comment (scopatz) [inline recipes/pyemcee/meta.yaml:28]: This should be matplotlib-base

File: recipes/pyemcee/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.4
+    - numpy >=1.15
+    - scipy >=1.2
+    - matplotlib >=2.2
```
[the remark this answers is not in the document]
Comment (danehkar) [reply recipes/pyemcee/meta.yaml]: @scopatz I got the building errors in **linux_64**, **osx_64**, and **win_64** after removing them from 'host'.

File: recipes/pyemcee/meta.yaml (hunk 1 above)
Comment (danehkar) [reply recipes/pyemcee/meta.yaml]: @scopatz I changed from **matplotlib** to **matplotlib-base**.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**